webapp/channels/src/components/admin_console/access_control/editors/cel_editor/editor.scss

```diff
@@ -0,0 +1,257 @@
+                background: linear-gradient(0deg, rgba(0, 0, 0, 0.16), rgba(0, 0, 0, 0.16)), var(--button-bg);
+            }
+        }
+    }
+
+    &__error-message {
+        display: flex;
+        align-items: center;
+
+        .icon {
+            margin-right: 4px;
+            font-size: 14px;
+        }
+    }
+
+    &__error {
+        display: flex;
+        align-items: center;
+        color: var(--button-color);
+    }
+
+    &__status-message {
+        display: flex;
+        align-items: center;
+
+        .icon {
+            margin-right: 4px;
+            font-size: 14px;
+
+            &.icon-refresh {
+                cursor: pointer;
+                &:hover {
+                    opacity: 0.8;
+                }
+            }
+        }
+    }
+
+    &__inline-validate-btn {
+        padding: 0;
+        border: none;
+        background: transparent;
+        color: var(--button-color);
+        cursor: pointer;
+        font-size: 12px;
+        font-weight: 600;
+
+        &:hover {
+            text-decoration: underline;
+        }
+
+        &:disabled {
+            cursor: not-allowed;
+            opacity: 0.6;
+        }
+
+        .cel-editor__loading {
+            display: flex;
+            align-items: center;
+
+            i {
+                margin-right: 4px;
+                font-size: 12px;
+            }
+        }
+    }
+
+    &__validate-btn {
+        display: none;
+    }
+}
+
+.cel-test-results-modal {
+    .modal-body {
+        max-height: 70vh;
+        overflow-y: auto;
+    }
+
+    .cel-test-attributes {
+        padding: 10px;
+        border-bottom: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+        margin-bottom: 20px;
+
+        .cel-attribute-tag {
+            display: inline-block;
+            padding: 4px 8px;
+            border-radius: 4px;
+            margin: 0 4px 4px 0;
+            background: rgba(var(--center-channel-color-rgb), 0.08);
+            font-size: 12px;
+        }
+    }
+
+    .cel-subjects-list {
+        .cel-subject-item {
+            padding: 12px;
+            border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+            border-radius: 4px;
+            margin-bottom: 12px;
+
+            .cel-subject-header {
+                margin-bottom: 8px;
+                font-weight: 600;
+            }
+
+            .cel-subject-attributes {
+                display: grid;
+                gap: 8px;
+                grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+
+                .cel-subject-attribute {
+                    .cel-attribute-key {
+                        margin-right: 4px;
+                        font-weight: 500;
+                    }
+
+                    .cel-attribute-value {
+                        color: rgba(var(--center-channel-color-rgb), 0.72);
+                    }
+                }
+            }
+        }
+    }
+}
+
+.Card__body.expanded:has(.cel-editor) {
```

Comment:
(Post-merge) Are you sure you nee this? I don't see anywhere that we're restricting the height of `.Card__body` elements anywhere